src/synthcity/plugins/core/plugin.py

```diff
@@ -548,8 +548,22 @@ def __init__(self, plugins: list, expected_type: Type, categories: list) -> None
         self._available_plugins = {}
         for plugin in plugins:
             stem = Path(plugin).stem.split("plugin_")[-1]
-            self._available_plugins[stem] = plugin
-
+            if stem == "goggle":
```

Comment:
Please don't include specific plugin conditions in `plugin.py`, this should stay generic.